.github/workflows/py-ci.yml

```diff
@@ -187,7 +187,8 @@ jobs:
             python -m pip install -e lambdas/shared
             python -m pip install -e lambdas/${{ matrix.path }}
           fi
-          python -m pip install -r lambdas/${{ matrix.path }}/requirements.txt -r lambdas/${{ matrix.path }}/test-requirements.txt
+          python -m pip install --no-deps -r lambdas/${{ matrix.path }}/requirements.txt
```

Comment:
I think it's important to spell out that without `--no-deps` second order dependencies can float and cause bugs and inconsistencies, not that anyone would read comments in the CI file. Maybe /lambdas needs a README with, for now, only that statement. Up to you.